dashboard/modules/snapshot/snapshot_head.py

```diff
@@ -89,6 +98,41 @@ async def snapshot(self, req):
             success=True, message="hello", snapshot=snapshot
         )
 
+    @routes.get("/api/component_activities")
+    async def get_component_activities(self, req) -> aiohttp.web.Response:
+        # Get activity information for driver
+        driver_activity_info = await self._get_job_activity_info()
+
+        resp = {"driver": dataclasses.asdict(driver_activity_info)}
+        return dashboard_optional_utils.rest_response(
```

Comment:
Why don't we just directly use `Response` like this? https://github.com/ray-project/ray/blob/8837a4593f0a3d817b4ad7bf111e5c40ab759d38/dashboard/modules/job/job_head.py#L70

`rest_response ` will change the output schema, and this is pretty vulnerable (imagine somebody changes the output schema of `rest_response `)